### DISCUSSION TITLE
What ID should a visitor submit ?

### DISCUSSION THREAD

**Discussion by KaKi87** [VirtusLab-Open-Source/strapi-plugin-comments#257] [Q&A]
Hi,

What ID should a visitor (account-less user) submit to `POST <host>/api/comments/api::<collection name>.<content type name>:<entity id>` ?

Thanks

**Comment by cyp3rius** [comment]
If you are using non-strapi user the id is just a string and it's up to your client web app to decide what you use. It might be e-mail or whatever unique by what you are going to identify user in your webapp.

**Reply by KaKi87** [reply]
Okay. Well, I was confused because when clicking one comment, all other comments from authors with different IDs also show up, so I thought it was mistakenly treating all comments as submitted by a single author.

But, if I understand corretly, it's rather grouping by post ?

Is there any way to group by author ?

Also, I'm not seeing the *pending* state shown in the README screenshot, in fact I'm not seeing the UI structured like in the screenshot at all, am I missing something ?

Thanks
